smac/tae/execute_ta_run.py

```diff
@@ -35,6 +35,18 @@ def enum_hook(obj):
                 return getattr(globals()[name], member)
         return obj
 
+class BudgetExhaustedException(Exception):
+    """ Exception indicating that time- or memory-budgets are exhausted. """
+    pass
+
+class TAEAbortException(Exception):
+    """ Exception indicating that the TAE suggests an ABORT of SMAC. """
```

Comment:
Corrected the doc-string. Would `AbortException` be enough? Or maybe `TargetAlgorithmAbortException`?